buddyboss/buddyboss-platform
# Fix deprecated warning for block category filter

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3037 
Fixes #3168

### How to test the changes in this Pull Request:

1. Go to WP Pages.
2. See warning is gone.

### Other information:
